README.md

```diff
@@ -41,6 +41,12 @@ The `default.config.yml` file has variables that must be customized before runni
 
 For Atom, you can customize packages to enable, install and disable using the Atom Package Manager tool `apm`. Add commands in `default.config.yml` under `atom_packages`. If you wish to configure the Atom editor styles, create a CSS/Less file and give Ansible the path to the file in `default.config.yml` under `atom_styling_path`.
 
+## Laptops
+
+- Make sure that you have the proper permissions for your user in place beforehand.
+- The laptops are only able to successfully setup the environment by changing PHPUnit to version 4.8 on [this line](https://github.com/chapmanu/ansible-workstation/blob/master/roles/phpunit/tasks/main.yml#L4).
+- If you receive an error regarding `rbenv rehash` then simply running the command manually and continuing from there solves the issue, but it doesn't happen consistently.
```

Comment:
I'd consider this an issue. So open an issue for it and if we can't resolve it, link to that.